packages/search/package.json

```diff
@@ -0,0 +1,51 @@
+{
+	"name": "@automattic/search",
+	"version": "1.0.0",
+	"description": "Automattic Search",
+	"homepage": "https://github.com/Automattic/wp-calypso",
+	"license": "GPL-2.0-or-later",
+	"author": "Automattic Inc.",
+	"main": "dist/cjs/index.js",
+	"module": "dist/esm/index.js",
+	"sideEffects": [
+		"*.css",
+		"*.scss"
+	],
+	"repository": {
+		"type": "git",
+		"url": "git+https://github.com/Automattic/wp-calypso.git",
+		"directory": "packages/search"
+	},
+	"publishConfig": {
+		"access": "public"
+	},
+	"bugs": {
+		"url": "https://github.com/Automattic/wp-calypso/issues"
+	},
+	"files": [
+		"dist",
+		"src"
+	],
+	"types": "dist/types",
+	"dependencies": {
+		"@automattic/viewport": "^1.0.0",
+		"@babel/runtime": "^7.11.1",
+		"@wordpress/components": "^10.0.5",
+		"@wordpress/i18n": "^3.14.0",
+		"@wordpress/icons": "^2.4.0",
+		"classnames": "^2.2.6",
+		"lodash": "^4.17.15"
+	},
+	"peerDependencies": {
+		"react": "^16.8",
+		"react-dom": "^16.8"
+	},
+	"devDependencies": {
+		"@storybook/addon-actions": "^5.3.18"
+	},
+	"scripts": {
+		"clean": "npx rimraf dist && tsc --build --clean",
+		"prepublish": "yarn run clean",
+		"prepare": "transpile && tsc && copy-assets"
```

Comment:
Hi @saramarcondes 👋 I noticed that even though the `search` package is written fully in TypeScript, the transpilation for the `dist` directories is done by Babel. The TypeScript compiler is used only to emit the type declaration.

Was this an intentional decision or is it some copy-paste accident? The compilation could be done 100% by the TypeScript compiler, with a pair of `tsconfig.json` and `tsconfig-cjs.json` config files. The `@automattic/data-stores` package does that, for example.

Reply:
This is an accident of copy and paste from the way the `components` package is configured. Please feel free to open a PR to fix this if you can, I’m not really familiar with how it would work without Babel...

Side note: the language picker package probably has the same problem.

Reply:
Thanks, I'll patch both packages in a PR :+1: